pages/api/module/post.ts

```diff
@@ -0,0 +1,37 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ErrorResponse } from "types/ErrorResponse";
+import { Module, ModuleInterface } from "../../../database/models/module";
+import { Tool } from "../../../database/models/tool";
+import connectDB from "../utils/mongoose";
+
+const post = async (
+    req: NextApiRequest,
+    res: NextApiResponse<ModuleInterface | ErrorResponse>,
+): Promise<void> => {
+    if (!req.body.title)
+        return res
+            .status(404)
```

Comment:
400 might make more sense than 404, since 404 usually means Page Not Found or File Not Found

Reply:
changes made.